miner_config/generate_config.py

```diff
@@ -54,25 +54,36 @@ def output_config_file(config, path):
     open(path, "w").write(config)
 
 
+def is_prodcution_fleet() -> bool:
+    return not bool(int(os.getenv('TESTNET', '0')))
+
+
+def is_device_type(board_name: str) -> bool:
+    return bool(int(os.getenv(board_name, '0')))
+
+
 def main():
     init_sentry()
-    if bool(int(os.getenv('PRODUCTION', '0'))):
+    if is_prodcution_fleet():
         base_url = 'https://helium-snapshots.nebracdn.com'
+        template_path = 'config.template'
     else:
         base_url = 'https://helium-snapshots-stage.nebracdn.com'
+        template_path = 'config-stage.template'
 
-    if bool(int(os.getenv('ROCKPI', '0'))):
+    if is_device_type('ROCKPI'):
         i2c_bus = 'i2c-7'
```

Comment:
In the future we can reference hardware definitions, but not important now: https://github.com/NebraLtd/hm-pyhelper/blob/b48f500b407d3c2ce8651844bb8a939746847691/hm_pyhelper/hardware_definitions.py#L20